tests/App/Resources/config/config.yml

```diff
@@ -0,0 +1,61 @@
+framework:
```

Comment:
In other PRs we are moving configuration from XML and YAML to PHP. Why should we do the opposite here?

Reply:
Here we should use PHP too. I copy yaml config becouse it was ready in other bundles. Convert this should be done in dedicated scope.